rotkehlchen/data_migrations/migrations/migration_10.py

```diff
@@ -24,14 +25,27 @@ def data_migration_10(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgress
     log.debug('Enter data_migration_10')
     with rotki.data.db.conn.read_ctx() as cursor:
         accounts = rotki.data.db.get_blockchain_accounts(cursor)
-    # steps are: ethereum accounts + potentially write to db + updating spam assets
-    progress_handler.set_total_steps(len(accounts.eth) + 3)
+    # steps are: ethereum accounts + potentially write to db + updating spam assets + polygon rpc
+    progress_handler.set_total_steps(len(accounts.eth) + 4)
 
     # Check updates for spam assets. This happens before accounts detection to avoid
     # detecting accounts that only have spam assets.
     progress_handler.new_step('Fetching new spam assets info')
     rotki.data_updater.check_for_updates(updates=[UpdateType.SPAM_ASSETS])
 
+    # we published a data version of the rpc nodes that contained a wrong etherscan name for
+    # polygon. this ensure that the name is correct both in the globaldb and the user db
```

Comment:
nitpick I did not see before, but don't restart for this

```suggestion
    # polygon. this ensures that the name is correct both in the globaldb and the user db
```